pyvista/core/filters/data_set.py

```diff
@@ -2698,12 +2697,17 @@ def streamlines_evenly_spaced_2D(dataset, vectors=None, start_position=None,
 
         Returns
         -------
-        streamlines : pyvista.PolyData
+        pyvista.PolyData
```

Comment:
```suggestion
        :class:`pyvista.PolyData`
```

With recent changes to our doc build, we have to explicitly call out the class reference to get the links working.